crates/sovereign-sdk/rollup-interface/src/state_machine/da.rs

```diff
@@ -15,10 +15,18 @@ use crate::BasicAddress;
 pub struct SequencerCommitment {
     /// Merkle root of soft confirmation hashes
     pub merkle_root: [u8; 32],
-    /// Start L1 block's hash
-    pub l1_start_block_hash: [u8; 32],
-    /// End L1 block's hash
-    pub l1_end_block_hash: [u8; 32],
+    /// Start L2 block's number
+    pub l2_start_block_number: u64,
+    /// End L2 block's number
+    pub l2_end_block_number: u64,
+    // /// Start L1 block's number
+    // pub l1_start_block_number: u64,
+    // /// End L1 block's number
+    // pub l1_end_block_number: u64,
```

Comment:
Do we need these comments

Reply:
The referenced github issue says that we should comment out code and not remove it.

Reply:
I mean while writing that I was just thinking about what steps to take... completely forgot about that, thought you would be removing them once the pr is finalized haha

Reply:
Removed